CAIPs/caip-xxx.md

```diff
@@ -0,0 +1,61 @@
+Title: Chain-Agnostic Wallet Address Request URI Scheme
+Type: Standard
+Status: Draft
+Authors: Tim Robinson
+Created: 2025-07-06
+
+## Abstract
+
+This CAIP introduces a chain-agnostic URI command (`web3://wallet_getAccounts`) designed to request an array of wallet addresses from any wallet application or service. It returns an array of standardised CAIP-10 format wallet addresses, which clearly specifies the blockchain namespace, chain reference, and wallet address.
```

Comment:
If this is an NFC-only or transport-agnostic protocol, the abstract should probably mention it!

And either way, a # Test Cases section (with both success cases and failure cases) would help, to make explicit what this query/return actually looks like as an NFC exchange, a JSON-RPC call/response, an HTTP GET/response, etc.

Reply:
@timjrobinson did you get any interest from implementers to take this over the line?  I feel like it's 99% of the way there, just a few gaps to make this worth publishing as a draft to leave a hook in the [more public] water[s]...